utils/mega_data_setup.py

```
@@ -351,33 +351,43 @@
                 DESIRED_ILVL_LIST.append(snipe_info)
 
         return DESIRED_ILVL_LIST
 
-    def __set_desired_ilvl(self, ilvl_info, item_names, base_ilvls):
+    def __set_desired_ilvl(
+        self, ilvl_info, item_names, base_ilvls, base_required_levels
+    ):
         if "item_ids" not in ilvl_info.keys():
             ilvl_info["item_ids"] = []
 
+        if "required_min_lvl" not in ilvl_info.keys():
```

Ruff

Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)